<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QWidget" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1343</width>
    <height>824</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1343</width>
    <height>824</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1343</width>
    <height>824</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="tabletTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>465</x>
     <y>140</y>
     <width>450</width>
     <height>390</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <italic>false</italic>
     <fontweight>Light</fontweight>
    </font>
   </property>
   <property name="text">
    <string>请打开图片</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>710</y>
     <width>921</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="D1">
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="previous">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>上一张(A)</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btn_open">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>打开图片(P)</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="next">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>下一张(D)</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="original_btn">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>原图(O)</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QScrollArea" name="R1">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>80</y>
     <width>371</width>
     <height>701</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>600</height>
    </size>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="RR1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>369</width>
      <height>1300</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>1300</height>
     </size>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>231</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="7" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>增强亮度</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="1">
       <widget class="QSlider" name="horizontalSlider_2">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSlider" name="horizontalSlider_4">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSlider" name="horizontalSlider_3">
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>增强锐度</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>增强对比度</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>增强颜色</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="horizontalSlider">
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="gridWidget_2" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>331</width>
       <height>1000</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>1000</height>
      </size>
     </property>
     <layout class="QGridLayout" name="RRR1_4">
      <item row="0" column="0">
       <widget class="QPushButton" name="mean_btn">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>均值滤波</string>
        </property>
       </widget>
      </item>
      <item row="15" column="0">
       <widget class="QPushButton" name="sketch_btn">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>素描</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QPushButton" name="reverse_btn">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>彩色负片</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="vena_btn">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>维纳滤波</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QPushButton" name="butter_high_btn">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>布特沃斯高</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="gamma_btn">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>伽马校正</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="inverse_btn">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>逆滤波</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="butter_low_btn">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>布特沃斯低</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QPushButton" name="gauss_low_btn">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>高斯低通</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="hist_btn">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>直方图均衡</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="lap_btn">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>拉普拉斯</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QPushButton" name="exp_btn">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>指数灰度</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QPushButton" name="gauss_high_btn">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>高斯高通</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QPushButton" name="edge_btn">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>边缘提取</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="fft_btn">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>傅立叶变换频谱</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="mid_btn">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>中值滤波</string>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QPushButton" name="ostu_btn">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Otsu</string>
        </property>
       </widget>
      </item>
      <item row="15" column="1">
       <widget class="QPushButton" name="oil_btn">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>油画</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="log_btn">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>对数灰度</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="power_btn">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>指数灰度</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>450</width>
     <height>390</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>5</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>原图</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>40</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>工具栏</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
